.changeset/witty-ducks-design.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-cloud-sdk/e2e-tests': patch
+'@sap-cloud-sdk/odata-common': patch
+'@sap-cloud-sdk/odata-v2': patch
+'@sap-cloud-sdk/odata-v4': patch
+---
+
+[Compatibility Note] Fix deserializer so it works in batch requests which only contain functions or actions. This changes the type parameters of `ActionFunctionImportRequestBuilderBase`.
```

Comment:
[req] And last point: We did not change anything in the (de-)serializers, therefore "fix deserializer" does not really describe what happened. My suggestion:

1. [Fixed Issue] Fix batch requests to accept only functions or actions.
2. [Compatibility Note] The `ActionFunctionImportRequestBuilderBase` has an additional mandatory type parameter to pass the according (de-)serializer type. This was missing previously due to a bug.

Of course feel free to chose better words. ;)

Reply:
I think that wording is very good. adapted.